app/views/admin/versions/_event_change_description.html.haml

```diff
@@ -0,0 +1,35 @@
+- if version.event == 'create'
```

Comment:
Okay I'm a bit confused.How do I use the simple form here? the conditions are complex and is not just a simple variable comparison